docs: grammar, spelling, and readability fixes (#3044)
fastify/fastify: docs/Recommendations.md

## Recommendations

> This document contains recommendations and best practices when using

Reviewer: ```suggestion
This document contains a set of recommendations when using
```

Can then move `Fastify.` to the same line and not exceed standard width.
